app/views/account/site_settings/edit.html.slim

```diff
@@ -9,11 +9,10 @@
     i.fa.fa-times-circle
 
 = simple_form_for :feature_flags, url: account_features_flags_path, method: :patch do |f|
-  - Flipper.features.each do |feature|
-    .feature-container
-      = f.input "#{feature.name}_enabled", as: :boolean, label: t("account.feature_flags.#{feature.name}.name"), input_html: { checked: feature.enabled? }
-      .description-feature
-        = feature.description
+  .feature-container
+    = f.input "new_calculator_design_enabled", as: :boolean, label: t("account.feature_flags.new_calculator_design.name"), input_html: { checked: Flipper[:new_calculator_design].enabled? }
+    .description-feature
+      = Flipper[:new_calculator_design].description
 

```

Comment:
why this code from previous PR is here?
you need to create new branch from develop, not from yours latest branch
if in previous branch will be changes you may forget to merge it to this 
fix it